### DISCUSSION TITLE
User guide: Target

### DISCUSSION THREAD

**Discussion by AVHopp** [emdgroup/baybe#42] [Documentation]
The user guide for targets needs an overhaul.

**Comment by Scienfitz** [comment]
My proposal for the content:
In general this will contain both info about targets and objective. Imo theres no reason to split these unless they have much more content each at some point.

- Targets and their role as representing measurements
- Objective and its role as telling baybe how to deal with multiple targets
- Targets
   - mention we only have the numerical target at this time
   - mention it can still be used to model ordinal categorical targets
   - it is limited and not suitable for non-ordinal categories
   - explain modes
   - explain optional bounds
   - explain optional transformation
- Objectives
   - Explain simple single target mode
   - Desirability
      - geom mean formula and scalarization approach/choices
      - explain requirements for bounds and transforms

**Reply by Scienfitz** [reply]
@AVHopp @AdrianSosic still would appreciate your blessing as I would like to write this still before xmas break

**Reply by AVHopp** [reply]
Thou hast mine blessing.

**Comment by AVHopp** [comment]
Closed since there is an ongoing PR implementing the Target/Objective User Guide.
